cmd/scaffold.go

```diff
@@ -1063,33 +1061,13 @@ func (fnb *FlowNodeBuilder) initDB() error {
 		return nil
 	}
 
-	// Pre-create DB path (Badger creates only one-level dirs)
+	// Pre-create DB path (pebble creates only one-level dirs)
 	err := os.MkdirAll(fnb.BaseConfig.datadir, 0700)
 	if err != nil {
 		return fmt.Errorf("could not create datadir (path: %s): %w", fnb.BaseConfig.datadir, err)
 	}
 
-	log := sutil.NewLogger(fnb.Logger)
-
-	// we initialize the database with options that allow us to keep the maximum
-	// item size in the trie itself (up to 1MB) and where we keep all level zero
-	// tables in-memory as well; this slows down compaction and increases memory
-	// usage, but it improves overall performance and disk i/o
-	opts := badger.
-		DefaultOptions(fnb.BaseConfig.datadir).
-		WithKeepL0InMemory(true).
-		WithLogger(log).
-
-		// the ValueLogFileSize option specifies how big the value of a
-		// key-value pair is allowed to be saved into badger.
-		// exceeding this limit, will fail with an error like this:
-		// could not store data: Value with size <xxxx> exceeded 1073741824 limit
-		// Maximum value size is 10G, needed by execution node
-		// TODO: finding a better max value for each node type
-		WithValueLogFileSize(128 << 23).
-		WithValueLogMaxEntries(100000) // Default is 1000000
```

Comment:
We have these find-tuned config for badger, however, I didn't do any for pebble, just used default options in this PR.